src/ansys/fluent/core/fluent_connection.py

```diff
@@ -250,7 +273,7 @@ def __init__(
                 port = os.getenv("PYFLUENT_FLUENT_PORT")
             self._channel_str = f"{ip}:{port}"
             if not port:
-                raise ValueError(
+                raise PortNotProvided(
                     "The port to connect to Fluent session is not provided."
```

Comment:
The patterns here are the same as in the other file. Why are you doing it this way?

Reply:
We will declare the error message in exception definition only. Thank you.

Reply:
Done. Thank you.